src/lib/Microsoft.Health.Fhir.Ingest.Template/CodeValueFhirTemplate.cs

```diff
@@ -14,6 +15,7 @@ public class CodeValueFhirTemplate : FhirTemplate
 #pragma warning restore CA2227
 
 #pragma warning disable CA2227
+        [JsonProperty(Required = Required.Always)]
         public virtual IList<FhirCode> Codes { get; set; }

```

Comment:
Ideally we would replace this with the CodeableConcept type and make that required which would mean they user would either need to supply the Text or at least one code element with in the codeable concept.  Unfortunately we can't just make that change without factoring in backwards compatibility.  

No actual action item, I am fine proceeding with this as an intermediate solution till we can support the full codeable concept in a backwards compatible manner.